alexandria/core/models.py

```diff
@@ -155,7 +155,8 @@ class Document(UUIDModel):
     date = models.DateField(blank=True, null=True)
 
     def get_latest_original(self):
-        return self.files.filter(variant=File.Variant.ORIGINAL).latest("created_at")
+        files = self.files.filter(variant=File.Variant.ORIGINAL)
+        return files.latest("created_at") if len(files) else None
```

Comment:
There are a few more call sites which will need to check for `None` now

Reply:
I've updated the other calls. I'm now raising a `ValidationError` everywhere code assumes that files must be present. Not sure if thats the right way though.

Reply:
Test coverage is failing, do you usually ignore Exceptions for coverage?

Reply:
I've added `no cover` to the check for now.

Reply:
Done